build_tools/customizations.rb

```diff
@@ -85,6 +85,9 @@ def dynamodb_example_deep_transform(subsegment, keys)
         members.delete('DestinationRegion')
         members.delete('PresignedUrl')
       end
+
+      # temporary
+      api['metadata']['protocols'] = ['ec2']
```

Comment:
I'm guessing this will be removed once services using ec2 protocol are using the new `protocols` metakey?